test/heex_formatter_test.exs

```diff
@@ -1138,4 +1138,26 @@ defmodule HeexFormatterTest do
 
     assert_formatter_output(input, expected)
   end
+
+  test "handle EEx comments" do
```

Comment:
Wrap the test on something like;

    if function_exported?(EEx, :tokenize, 2) do

:)